### PR TITLE
Simplify DeepSpeech Windows worker setup

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: worker_types/deepspeech-win/userdata [hunk 1]
```diff
@@ -27,47 +27,35 @@ Set-ExecutionPolicy Unrestricted -Force -Scope Process
...
+# install golang
+md "C:\gopath"
 
-# install more rustc dependencies (64 bit)
-$client.DownloadFile("http://download.microsoft.com/download/4/c/b/4cbd5757-0dd4-43a7-bac0-2a492cedbacb/vc_redist.x64.exe", "C:\vcredist_x64-vs2015.exe")
-Start-Process "C:\vcredist_x64-vs2015.exe" -ArgumentList "/install /passive /norestart /log C:\vcredist_x64-vs2015-install.log" -Wait -PassThru
+# set permanent env vars
+[Environment]::SetEnvironmentVariable("GOROOT", "C:\go", "Machine")
+[Environment]::SetEnvironmentVariable("PATH", $Env:Path + ";C:\gopath\bin", "Machine")
+[Environment]::SetEnvironmentVariable("GOPATH", "C:\gopath", "Machine")
```
Comment (petemoore) [inline worker_types/deepspeech-win/userdata]: Note, you only need to install go if you are using it in your tasks. It isn't needed by generic-worker, so you can remove lines 39-45 if you want.

File: worker_types/deepspeech-win/userdata [hunk 2]
```diff
@@ -76,12 +64,6 @@ $ace = New-Object System.Security.AccessControl.FileSystemAccessRule("Everyone",
 $acl.AddAccessRule($ace)
 Set-Acl "C:\builds" $acl
 
-# download tooltool
-$client.DownloadFile("https://raw.githubusercontent.com/mozilla/build-tooltool/master/tooltool.py", "C:\builds\tooltool.py")
-
-# install nssm
-Expand-ZIPFile -File "C:\nssm-2.24.zip" -Destination "C:\" -Url "http://www.nssm.cc/release/nssm-2.24.zip"
-
```
Comment (petemoore) [inline worker_types/deepspeech-win/userdata]: You need to keep nssm, it is needed by generic-worker.

File: worker_types/deepspeech-win/userdata [hunk 3]
```diff
@@ -101,58 +83,17 @@ Set-Content -Path "C:\Windows\System32\drivers\etc\hosts" -Value $HostsFile_Cont
...
-# upgrade python libraries
-Start-Process "C:\mozilla-build\python\python.exe" -ArgumentList "-m pip install --upgrade pip==8.1.1 setuptools==20.7.0 virtualenv==15.0.3 wheel==0.29.0 pypiwin32==219 requests==2.9.1 psutil==4.1.0" -Wait -NoNewWindow -PassThru -RedirectStandardOutput "C:\python_package_upgrades.log" -RedirectStandardError "C:\python_package_upgrades.err"
-
-# set env vars for the currently running process
-$env:GOROOT = "C:\go"
-$env:GOPATH = "C:\gopath"
-$env:PATH   = $env:PATH + ";C:\go\bin;C:\gopath\bin;C:\mozilla-build\python;C:\mozilla-build\python\Scripts;C:\Program Files\Git\cmd"
-
 # get generic-worker and livelog source code (note required, but useful)
 Start-Process "go" -ArgumentList "get -t github.com/taskcluster/generic-worker github.com/taskcluster/livelog" -Wait -NoNewWindow -PassThru -RedirectStandardOutput "C:\generic-worker\go-get_install.log" -RedirectStandardError "C:\generic-worker\go-get_install.err"
```
Comment (petemoore) [inline worker_types/deepspeech-win/userdata]: you can remove lines 97/98 - only the binary of livelog/taskcluster-proxy is needed, the source code isn't used by anything

File: worker_types/deepspeech-win/userdata (hunk 1 above)
Comment (lissyx) [reply worker_types/deepspeech-win/userdata]: But the `generic-worker` and `livelog` install after makes use of it, no?

File: worker_types/deepspeech-win/userdata (hunk 2 above)
Comment (lissyx) [reply worker_types/deepspeech-win/userdata]: done, thanks

File: worker_types/deepspeech-win/userdata (hunk 3 above)
Comment (lissyx) [reply worker_types/deepspeech-win/userdata]: okay, it's consistent with the comment above

File: worker_types/deepspeech-win/userdata (hunk 1 above)
Comment (lissyx) [reply worker_types/deepspeech-win/userdata]: nevermind, just understood

**Comment by lissyx** [comment]
@petemoore With the installation of GNUtar instead of BSDtar, it should now be all okay to be merged :)

**Comment by petemoore** [comment]
Note, the CI failures are unrelated, and I'll handle those in a separate PR.

**Pull request merged**